docs/index.rst

```diff
@@ -11,9 +11,11 @@ toqito focuses on providing numerical tools to study problems pertaining to
 entanglement theory, nonlocal games, and other aspects of quantum information
 that are often associated with computer science.
 
-.. image:: http://img.shields.io/travis/vprusso/toqito.svg?style=plastic
+[![build status](
```

Comment:
This line shows up as a typo. 

![image](https://github.com/vprusso/toqito/assets/66048318/9dbca5e3-dd5b-44f5-a7c1-8c047afcbfe5)

Reply:
Should be fixed by https://github.com/vprusso/toqito/pull/262/commits/5ece7a9389daf8e793fe3e49a2a277ef3af5efac